src/heartbeat.ts

```diff
@@ -139,6 +139,8 @@ export class HeartbeatManager extends Disposable {
                 } else {
                     this.logService.trace('Stopping heartbeat as workspace is not running');
                     this.stopHeartbeat();
+                    this.stopEventCounter();
```

Comment:
When we stop, we should report the final analytics event which reports the final delta.

I think we should also merge `vscode_desktop_heartbeat_delta` into `ide_heartbeat` by adding more context in case of VS Code Desktop.